src/options.ts

```diff
@@ -1,8 +1,4 @@
 import { CreateListenerOptions, DialOptions } from '@libp2p/interface-transport'
 
-export interface WebRTCListenerOptions extends CreateListenerOptions {
-  //, WebRTCInitiatorInit {
-  //   channelOptions?: WebRTCReceiverInit
-}
-
+export interface WebRTCListenerOptions extends CreateListenerOptions {}
```

Comment:
my linters do not accept such empty extends, I believe better practice is something like:

```
export { CreateListenerOptions as WebRTCListenerOptions } from '@libp2p/interface-transport'
```